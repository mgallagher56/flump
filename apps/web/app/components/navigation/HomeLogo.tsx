import { Link } from '@remix-run/react';

import logo from '/images/flump_logo.png';

const HomeLogo = () => {
  return (
    <div className="home-logo">
      <Link to="/">
        <img alt="flump logo" height={50} src={logo} width={50} />
      </Link>
    </div>
  );
};

export default HomeLogo;
