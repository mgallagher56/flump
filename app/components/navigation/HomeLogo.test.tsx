import { render } from '@testing-library/react';
import { describe, expect, test } from 'vitest';
import HomeLogo from '~/components/navigation/HomeLogo';

describe('<HomeLogo />', () => {
  test('renders as expected', () => {
    const { baseElement } = render(<HomeLogo />);
    expect(baseElement).toMatchSnapshot();
  });
});
