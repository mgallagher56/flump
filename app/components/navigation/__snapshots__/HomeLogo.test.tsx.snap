// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`<HomeLogo /> > renders as expected 1`] = `
<body>
  <div>
    <div
      class="home-logo"
    >
      <img
        alt="flump logo"
        height="50"
        src="/public/images/flump_logo.png"
        width="50"
      />
    </div>
  </div>
</body>
`;
